cmp/grid/GridModel.js

```diff
@@ -697,6 +699,17 @@ export class GridModel {
         if (colIds.length) this.agColumnApi.autoSizeColumns(colIds);
     }
 
+    /**
+     * Set a flag acknowledging that a framework component has been mounted.
+     * We debounce here catch the end of a collection of components mounted at
+     * the same time (i.e. column header initialisation)
+     */
+    @debounced(50)
+    @action
+    noteFrameworkCmpMounted() {
+        this.frameworkCmpsMounted = true;
```

Comment:
why do we want/need to wait/delay until they are all mounted?  Doesn't this just become a no-op for the subsequent calls?